### PR TITLE
added active class to self._button$ element

### PR DESCRIPTION
**Pull request by bbhagwandeen**
Added the activate class to element self._button @flukeout 

out

### CODE REVIEW

**Comment by flukeout** [comment]
Testing it out now, thanks!

File: public/editor/scripts/editor/js/fc/bramble-popupmenu.js [hunk 1]
```diff
@@ -22,6 +22,8 @@ define(function(require) {
     self.close = function(e) {
       if(e) {
         e.stopPropagation();
+        self._button$.removeClass('active');
+        $("#editor-pane-nav-options").css("opacity", "0.5");
```
Comment (flukeout) [inline public/editor/scripts/editor/js/fc/bramble-popupmenu.js]: Now that we are adding and removing the ``active`` class to the element, we can get rid of the code that changes the CSS and instead add some styles to the ``editor.less`` file [here](https://github.com/mozilla/thimble.mozilla.org/blob/master/public/editor/stylesheets/editor.less#L351)

Like this...
```
#editor-pane-nav-options.active,
#editor-pane-nav-options:hover {
    opacity: 0.7;
    filter: alpha(opacity=70);
}
```

**Comment by flukeout** [comment]
Okay, great stuff - left a comment up above about how to improve this.

**Comment by flukeout** [comment]
Hey @bbhagwandeen - just pinging you so you're aware I left some comments, thanks!

**Comment by bbhagwandeen** [comment]
Hey @flukeout . Thanks for the comments! I'll update it ASAP. I was trying to do it that way but didn't know how to write it using the propey CSS syntax.

**Comment by flukeout** [comment]
Thanks, looking forward to it @bbhagwandeen

File: public/editor/scripts/editor/js/fc/bramble-popupmenu.js (hunk 1 above)
Comment (bbhagwandeen) [inline public/editor/scripts/editor/js/fc/bramble-popupmenu.js]: Ok. With that addition, I can remove the `$("#editor-pane-nav-options").css("opacity", "0.5");`
and `$("#editor-pane-nav-options").css("opacity", "0.7");` . I still have to use `.addClass('active')` and `.removeClass('active')` with `self._button$`, however. Is that the right way? If not, let me know and i'll figure it out!

**Comment by flukeout** [comment]
Testing again now, thank you!

File: public/editor/scripts/editor/js/fc/bramble-popupmenu.js [hunk 2]
```diff
@@ -48,8 +49,9 @@ define(function(require) {
     // Toggle the menu on/off when the button is clicked.
     self._button$.on("click", function(e) {
       e.stopPropagation();
-
+
```
Comment (flukeout) [inline public/editor/scripts/editor/js/fc/bramble-popupmenu.js]: Please get rid of the whitespace here and we can merge this in!

**Comment by flukeout** [comment]
Hey @bbhagwandeen this works great, I left one final request to get rid of the whitespace that was introduced into the file. In your editor you can find an option that shows whitespace which is handy.

Thanks!

**Comment by flukeout** [comment]
Great, thank you @bbhagwandeen! Good work 👍

**Pull request merged**